bluej/src/main/java/bluej/utility/javafx/DelegableScalableTextField.java

```diff
@@ -315,11 +430,56 @@ public void changed(ObservableValue<? extends Number> observable, Number oldValu
         addEventHandler(KeyEvent.KEY_PRESSED, e -> { if (e.getCode() == KeyCode.ESCAPE) delegate.escape(); });
         
     }
+
+    @OnThread(Tag.FXPlatform)
+    private void handleInputMethodEvent(InputMethodEvent event) {
+        // This is adapted from the default handler in TextInputControlSkin.  That handler
+        // uses selection to keep track of some things.  We keep track manually here but
+        // there is not currently a visual representation of the highlight.
```

Comment:
I don't understand this very last sentence - is it something that the default method would do but we don't (and does it matter?)